docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -70,7 +71,7 @@ Checklist:
 - [ ] **Stage 1 - Internal Testing**
   - [ ] CHANGELOG.md has been updated
     - use [`./bin/mkreleaselog`](https://github.com/ipfs/go-ipfs/tree/master/bin/mkreleaselog) to generate a nice starter list
```

Comment:
How I make it works:

- install zsh
- GO111MODULE=off go get -u github.com/ipfs/kubo
- Go to $GOPATH/src/ipfs/kubo and execute `./bin/mkreleaselog`